packages/react-native-client/package.json

```diff
@@ -0,0 +1,115 @@
+    "lint": "eslint . --ext .ts,.tsx --fix",
+    "lint:check": "eslint . --ext .ts,.tsx",
+    "test": "expo-module test",
+    "prepare": "expo-module prepare",
+    "prepublishOnly": "expo-module prepublishOnly",
+    "expo-module": "expo-module",
+    "release": "release-it",
+    "releaseAppAndroid": "cd examples/video-chat/android && fastlane deploy",
+    "releaseAppIos": "cd example/ios && fastlane deploy",
+    "docs": "sed 's/\\.github\\/images/media/g' README.md > README_docs.md && typedoc src/index.tsx --media ./.github/images --readme README_docs.md && rm README_docs.md",
+    "typecheck": "yarn tsc --noEmit && cd example && yarn tsc --noEmit"
+  },
+  "keywords": [
+    "react-native",
+    "expo",
+    "@fishjam-dev/react-native-client",
+    "MembraneWebRTC",
+    "Fishjam"
+  ],
+  "repository": "https://github.com/fishjam-dev/react-native-client-sdk",
+  "author": "Software Mansion (https://swmansion.com)",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/fishjam-dev/react-native-client-sdk/issues"
+  },
+  "homepage": "https://github.com/fishjam-dev/react-native-client-sdk#readme",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/",
+    "access": "public"
+  },
+  "dependencies": {
+    "phoenix": "^1.7.6",
+    "promise-fs": "^2.1.1",
+    "protobufjs": "^7.2.4"
+  },
+  "devDependencies": {
+    "@testing-library/react": "^14.0.0",
+    "@types/lodash": "^4.14.195",
+    "@types/phoenix": "^1.6.0",
+    "@types/promise-fs": "^2.1.2",
+    "@types/react": "^18.0.25",
+    "expo": "^50.0.17",
+    "expo-module-scripts": "^3.0.4",
+    "expo-modules-core": "~1.11.8",
+    "husky": "^9.0.10",
+    "jest": "^29.6.1",
+    "jest-environment-jsdom": "^29.6.1",
+    "jest-expo": "^50.0.2",
+    "jest-websocket-mock": "^2.4.0",
+    "lint-staged": "^15.2.2",
+    "pod-install": "^0.2.0",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-native": "0.73.4",
+    "release-it": "^17.0.3",
+    "ts-proto": "^1.176.0",
+    "typedoc": "^0.25.7",
+    "typedoc-plugin-mark-react-functional-components": "^0.2.2",
+    "typescript": "^5.4.0"
+  },
+  "peerDependencies": {
+    "expo": "*",
+    "react": "*",
+    "react-native": "*"
+  },
+  "jest": {
+    "preset": "jest-expo",
+    "modulePathIgnorePatterns": [
+      "<rootDir>/example/node_modules",
+      "<rootDir>/lib/"
+    ],
+    "testEnvironment": "jsdom"
+  },
+  "eslintConfig": {
+    "root": true,
+    "extends": [
+      "@react-native-community",
+      "prettier"
+    ],
+    "settings": {
+      "import/resolver": {
+        "babel-module": {}
+      }
+    },
+    "rules": {
+      "prettier/prettier": [
+        "error",
+        {
+          "quoteProps": "consistent",
+          "singleQuote": true,
+          "tabWidth": 2,
+          "trailingComma": "es5",
+          "useTabs": false
+        }
+      ]
+    }
+  },
+  "lint-staged": {
+    "*.(js|ts|tsx)": [
+      "yarn format:check",
+      "yarn lint:check"
```

Comment:
Is there a reason why those comands are not `"yarn format"`, and `"yarn lint"`

Reply:
Yes, there is a reason. I don't want to modify any files on this stage. Modified files will not be added to `staged changes` and thus will not be committed together. I would be better to just stop commit and ask user to manually fix all issues